content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,378 @@
+
+Configuration in Odoo
+=====================
+
+Before using AvaTax there are some additional configurations in Odoo to ensure tax calculations are
+made accurately.
+
+Verify that the Odoo database contains necessary data. The country initially set up in the database
+determines the fiscal position and aids AvaTax in calculating accurate tax rates.
+
+Fiscal country
+--------------
+
+
+To set the :guilabel:`Fiscal Country`, navigate to :menuselection:`Accounting app --> Configuration
+--> Settings`.
+
+.. seealso::
+   - :doc:`../../fiscal_localizations`
+
+Under the :guilabel:`Taxes` section, set the :guilabel:`Fiscal Country` feature to :guilabel:`United
+States` or :guilabel:`Canada`.
+
+Company settings
+----------------
+
+All companies operating under the Odoo database should have a full and complete address listed in
+the settings. Navigate to :menuselection:`Settings App, and under the :guilabel:`Companies` section,
+ensure there is only one company operating the Odoo database. Click :guilabel:`Update Info` to open
+a separate page to update company details.
+
+Should there be multiple companies operating in the database, click :guilabel:`Manage Companies` to
+load a list of companies to select from. Update customer information by clicking into the specific
+customer.
+
+Database administrators should ensure that the :guilabel:`Street...`, :guilabel:`Street2...`,
+:guilabel:`City`, :guilabel:`State`, :guilabel:`ZIP`, and :guilabel:`Country` are all updated for
+the companies.
+
+This will ensure accurate tax calculations and smooth end-of-year accounting operations.
+
+.. seealso::
+   - :doc:`../../../general/companies`
+   - :doc:`../get_started`
+
+Module installation
+-------------------
+
+Next, ensure that the Odoo *AvaTax* module is installed. To do so, navigate to :menuselection:`Apps
+application. In the :guilabel:`Search...` bar, type in `avatax`. Press enter and the following
+results populate:
+
+- :guilabel:`Avatax`: `account_avatax`
+- :guilabel:`Avatax for SO`: `account_avatax_sale`
+- :guilabel:`Avatax for Subscriptions`: `account_avatax_sale_subscription`
+- :guilabel:`Avatax Brazil`: `l10n_br_avatax`
+- :guilabel:`Avatax for SOs in Brazil`: `l10n_br_avatax_sale`
+- :guilabel:`Account Avatax - Ecommerce`: `website_sale_account_avatax`
+- :guilabel:`Account AvaTax - Ecommerce - Delivery`: `website_sale_delivery_avatax`
+
+Click on the :guilabel:`Install` button on the module labeled :guilabel:`Avatax`: `account_avatax`.
+This will install the following modules:
+
+- :guilabel:`Avatax`: `account_avatax`
+- :guilabel:`Avatax for SO`: `account_avatax_sale`
+- :guilabel:`Account Avatax - Ecommerce`: `website_sale_account_avatax`
+
+Should *AvaTax* be needed for Odoo *Subscriptions* or for delivery tax in Odoo *Ecommerce*, then
+install those modules individually by clicking on :guilabel:`Install`. Additionally, in Brazil, two
+separate *AvaTax* modules need to be installed. See the list above for these modules.
```

Comment:
Suggestion to format this modules list similar to how the Fiscal Localizations do it, take Colombia for example:
https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/colombia.html#modules-installation